extension/{{cookiecutter.out_dir}}/docs/index.md

```diff
@@ -4,6 +4,29 @@
 
 ## Examples
 
+```
+import flet as ft
+
+from {{cookiecutter.project_name_underscore}} import {{cookiecutter.control_name}}
+
+
+def main(page: ft.Page):
+    page.vertical_alignment = ft.MainAxisAlignment.CENTER
+    page.horizontal_alignment = ft.CrossAxisAlignment.CENTER
+
+    page.add(
+
+                ft.Container(height=150, width=300, alignment = ft.alignment.center, bgcolor=ft.Colors.PURPLE_200, content={{cookiecutter.control_name}}(
+                    tooltip="My new {{cookiecutter.control_name}} Control tooltip",
+                    value = "My new {{cookiecutter.control_name}} Flet Control", 
+                ),),
+
+    )
+
+
+ft.app(main)
+```
```

Comment:
This example will not apply to all controls (example, non-visible controls).
I suggest we have a simple/basic & static example. (for ex: Hello World or Counter)